collect_app/src/androidTest/java/org/odk/collect/android/tasks/DownloadFormListTaskTest.java

```diff
@@ -14,6 +14,35 @@
 
 public class DownloadFormListTaskTest extends MockedServerTest {
 
+    private static final String RESPONSE = join(
```

Comment:
This change I disagree pretty strongly with. It seems to make things harder to read for no real benefit. What do you think, @shobhitagarwal1612?

Reply:
I feel like if the response was in a file within `assets` folder, then it would have been much neater.